src/config-loader/config-loader.ts

```diff
@@ -282,14 +310,36 @@ export const setup = (app: Express.Application) => {
 				}),
 			);
 			await loadConfig(configObj);
+			await loadAsyncMigrations(configObj);
 		} catch (err) {
 			console.error('Error loading application config', err, err.stack);
 			process.exit(1);
 		}
 	};
 
+	const loadAsyncMigrations = async (data: Config): Promise<any> => {
+		try {
+			await Promise.all(
+				data.models.map(async (model) => {
+					if (
+						model.migrations != null &&
+						Object.keys(model.migrations).includes(migrationCategories.async)
+					) {
+						await sbvrUtils.executeAsyncModelMigration(
+							model as sbvrUtils.ExecutableModel,
+						);
```

Comment:
If migrations run until shutdown, will this promise ever resolve?

Reply:
The promise resolved as intervals were setup inside of it. Still it confuses and I've taken out the async await of this call and the (for await) loop inside the async migrator. All async migrations must not interfer with other async migrations executed in one deployment, thus the setup of it is save to be not async/await.
Thx!